packages/maskbook/src/utils/native-rpc/types.ts

```diff
@@ -0,0 +1,27 @@
+import type * as settings from '../../settings/settings'
+/**
+ * JSON RPC calls that can be called from the native side.
+ * JS = server, native = client
+ */
+export interface WebviewAPIs {
+    web_echo<T>(arg: T): Promise<T>
+    getDashboardUrl(path: string): Promise<string>
+    getSettings(key: keyof typeof settings): Promise<any>
+    getConnectedPersonas(): Promise<{ network: string; connected: boolean }[][]>
```

Comment:
Hmm, why it is `{...}[][]` instead of `{...}[]`?

Reply:
The api returns `[[{network: 'facebook.com', connected: false}, {network: 'twitter.com', connected: true}],[//...],[//...]] `

Reply:
So what difference between `result[0]` and `result[1]`? Are they belong to different personas? If so why there isn't any thing to mark it?

Reply:
They belong to different personas. @jk234ert told me it is okay even there's nothing to mark it.